framework/global/stringutils.cpp

```diff
@@ -27,3 +30,12 @@ bool mu::strings::replace(std::string& str, const std::string& from, const std::
     str.replace(start_pos, from.length(), to);
     return true;
 }
+
+std::string mu::strings::toLower(const std::string& source)
```

Comment:
doesn't the standard library already provide these functions?

Reply:
Not. Can only do: 
```
    std::for_each(str.begin(), str.end(), [](char& c) {
        c = ::tolower(c);
    });
```
or
```
std::transform(s.begin(), s.end(), s.begin(),   [](unsigned char c){ return std::tolower(c); }  );
```
Therefore, it’s easier to add a function